.github/workflows/pr-checker.yaml

```diff
@@ -0,0 +1,37 @@
+name: pr-checker
+on: 
+  pull_request:
+env:
+  GO_VERSION: "1.16"
+jobs:
+  build-test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+        matrix:
+            os: [ubuntu-20.04]
+    steps:
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - name: Checkout Code
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Install Dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install --yes build-essential pkgconf libelf-dev llvm-12 clang-12
+          for tool in "clang" "llc" "llvm-strip"
+          do
+            sudo rm -f /usr/bin/$tool
+            sudo ln -s /usr/bin/$tool-12 /usr/bin/$tool
+          done
+          sudo curl -L -o /usr/bin/opa https://github.com/open-policy-agent/opa/releases/download/v0.35.0/opa_linux_amd64 && sudo chmod 755 /usr/bin/opa
+      - name: Build
+        run: make all
+      - name: Test tracee-ebpf
```

Comment:
We're changing CI workflow to test compilation while @rafaeldtinoco is working on Makefiles refactor.

Reply:
Yep, this allows us to have CI tests while changing Makefiles (as the only thing we need to guarantee on further commits is that "make all" and "make test" works).

Reply:
I decided to step back from new workflows in favour or fixing build in container (with DOCKER=1)